data/json/martialarts.json

```diff
@@ -844,13 +878,17 @@
         "name" : "Lizard Kung Fu",
         "description": "One of the Five Deadly Venoms, and likely to be refactored to a mutation at the next save-compatibility breaker. Lizard Style focuses on using walls to your advantage.",
         "arm_block" : 2,
+        "leg_block" : 50,
         "techniques" : [
         ]
+        //lizard is not implented
```

Comment:
Comma after the bracket closure, too, I'd think, since this makes the closure no longer the final entry?